/bin/bash: /sw/ubuntu-22.04/anaconda3/current/lib/libtinfo.so.6: no version information available (required by /bin/bash)
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Half-Cheetah-RM1-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Half-Cheetah-RM2-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M0-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M1-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M2-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M3-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M4-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M5-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M6-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M7-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M8-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M9-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-M10-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M0-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M1-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M2-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M3-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M4-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M5-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M6-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M7-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M8-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M9-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Water-single-M10-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Office-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Office-single-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M0-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M1-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M2-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M3-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M4-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M5-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M6-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M7-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M8-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M9-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-M10-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M0-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M1-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M2-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M3-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M4-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M5-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M6-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M7-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M8-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M9-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Craft-single-M10-v0[0m
  logger.warn("Overriding environment {}".format(id))
/u/hbt9su/.local/lib/python3.6/site-packages/gym/envs/registration.py:216: UserWarning: [33mWARN: Overriding environment Taxi-v0[0m
  logger.warn("Overriding environment {}".format(id))
WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/tf_util.py:53: The name tf.get_default_session is deprecated. Please use tf.compat.v1.get_default_session instead.

WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/tf_util.py:63: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/tf_util.py:70: The name tf.InteractiveSession is deprecated. Please use tf.compat.v1.InteractiveSession instead.

2024-01-21 14:08:38.085277: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2024-01-21 14:08:38.112924: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Could not dlopen library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /u/hbt9su/.local/lib/python3.6/site-packages/cv2/../../lib64:/sw/ubuntu-22.04/cudnn/8.1.0/lib:/sw/ubuntu-22.04/cuda/current/extras/CUPTI/lib64:/sw/ubuntu-22.04/cuda/current/lib64:/sw/ubuntu-22.04/anaconda3/current/lib::/usr/lib/nvidia:/u/hbt9su/.mujoco/mujoco210/bin:/usr/lib/nvidia:/u/hbt9su/.mujoco/mujoco210/bin
2024-01-21 14:08:38.112968: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
2024-01-21 14:08:38.112993: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (cortado06): /proc/driver/nvidia/version does not exist
2024-01-21 14:08:38.435814: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200000000 Hz
2024-01-21 14:08:38.436216: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56113f21cff0 executing computations on platform Host. Devices:
2024-01-21 14:08:38.436257: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/misc_util.py:58: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

WARNING:tensorflow:From /u/hbt9su/PartialSatLTL/psltl/baseline_algo/crm/rl_agents/dhrm/controller.py:84: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

WARNING:tensorflow:From /u/hbt9su/PartialSatLTL/psltl/baseline_algo/crm/rl_agents/dhrm/build_graph.py:130: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/input.py:57: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
WARNING:tensorflow:From /u/hbt9su/baselines/baselines/common/models.py:94: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c72508d0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c72508d0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c7250e80>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c7250e80>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faf978>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faf978>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faf978>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faf978>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faffd0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faffd0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faffd0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6faffd0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:From /u/hbt9su/PartialSatLTL/psltl/baseline_algo/crm/rl_agents/dhrm/build_graph.py:145: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ef8c88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ef8c88>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6eb0828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6eb0828>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b390>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b390>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c726dc18>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c726dc18>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b0b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b0b8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b0b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6f9b0b8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ea2e80>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ea2e80>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6e4df60>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6e4df60>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bcc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bcc0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bcc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bcc0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bc88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bc88>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bc88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e5bc88>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ea2400>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6ea2400>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6b64d30>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6b64d30>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e54860>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e54860>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b9b630>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b9b630>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b57be0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b57be0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e543c8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6e543c8>>: AttributeError: module 'gast' has no attribute 'Index'
2024-01-21 14:08:40.212633: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6393c18>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c6393c18>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c63b77b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c63b77b8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63b27b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63b27b8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63a49b0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63a49b0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63b27b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63b27b8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63bc5f8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c63bc5f8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ff470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ff470>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62b3160>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62b3160>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3f28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3f28>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3f28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3f28>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3048>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b3048>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62a4f28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62a4f28>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62eea58>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62eea58>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ba828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ba828>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6259940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6259940>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b18470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c6b18470>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5fd9f28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5fd9f28>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b36d8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c62b36d8>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62a4ef0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62a4ef0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ba7f0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7ff5c62ba7f0>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72940>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72940>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72a20>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72a20>>: AttributeError: module 'gast' has no attribute 'Index'
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72a20>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7ff5c5f72a20>>: AttributeError: module 'gast' has no attribute 'Index'
/var/spool/slurmd/job2794113/slurm_script: line 53: 3891597 CPU time limit exceeded (core dumped) python run.py --alg ${env_name} --env ${env} --num_timesteps ${timesteps} --gamma ${gamma} --env_name ${env_name} --seed ${seed} --use_rs
